From f1ef03abc936274fc2f01ac758f3211d712e40f7 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Tue, 15 Dec 2020 16:58:40 -0800
Subject: [PATCH 104/115] SystemUI: Skip screen-on animation in all scenarios

Unlike turning the screen off, turning it on is a time-critical
operation that should be done as fast as possible without a slow
animation. It's not only noticeable when using capacitive fingerprint
sensors.

Change-Id: I87276128456afd9f535742477d9f76d025a12539
---
 .../src/com/android/systemui/statusbar/phone/StatusBar.java    | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
index a36f8c6613c..8ae05ded63b 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
@@ -4657,8 +4657,7 @@ public class StatusBar extends SystemUI implements DemoMode,
 
         boolean launchingAffordanceWithPreview =
                 mNotificationPanelViewController.isLaunchingAffordanceWithPreview();
-        mScrimController.setLaunchingAffordanceWithPreview(launchingAffordanceWithPreview
-                || mBiometricUnlockController.isWakeAndUnlock());
+        mScrimController.setLaunchingAffordanceWithPreview(true);
 
         if (mBouncerShowing) {
             // Bouncer needs the front scrim when it's on top of an activity,
-- 
2.25.1

